bokehjs/src/coffee/source/ajax_data_source.coffee

```diff
@@ -0,0 +1,49 @@
+define [
+  "underscore"
+  "backbone"
+  "common/logging"
+  "./remote_data_source"
+], (_, Backbone, Logging, RemoteDataSource) ->
+
+  logger = Logging.logger
+
+  #maybe generalize to ajax data source later?
+  class AjaxDataSource extends RemoteDataSource.RemoteDataSource
+    type: 'AjaxDataSource'
+    destroy : () =>
+      if @interval?
+        clearInterval(@interval)
+
+    setup : (plot_view, glyph) =>
+      @pv = plot_view
+      @update()
+      if @get('polling_interval')
+        @interval = setInterval(@update, @get('polling_interval'))
+
+    update : () =>
+      $.ajax(
+        dataType: 'json'
```

Comment:
Let's get a little experience with this first then we can extend it with more knobs and dials later.